README.md

```diff
@@ -19,20 +18,96 @@ It also supports the diagnostic metrics:
 
-<h4 id="install-master">Install from master</h4>
+Refer to the [official DevTools Performance panel documentation](https://developer.chrome.com/docs/devtools/performance/overview) for information about using each feature. As of Chrome 132 in January 2025, here is a quick summary of the extension features supported by DevTools:
+
+#### Live metrics
+
+When you open the Performance panel, the view defaults to showing a live look at your local Core Web Vitals metrics.
+
+![LCP, CLS, and INP live metrics in DevTools](media/devtools-live-metrics.png)
+
+#### Field data
+
+Use the "Field data" configuration flow to show field data from CrUX next to your local metrics. URL and origin-level data is available for both desktop and mobile.
+
+![Field data in DevTools](media/devtools-field-data.png)
+
+#### LCP element attribution
+
+You can find a reference to the element responsible for the LCP metric at the bottom of the LCP section.
+
+![LCP attribution in DevTools](media/devtools-lcp-attribution.png)
+
+#### LCP phases
+
+Hover over the LCP metric values to display a card with more information about your local and field LCP performance, including a breakdown of your local LCP phases: TTFB, load delay, load duration, and render delay.
+
+![LCP phases in DevTools](media/devtools-lcp-phases.png)
+
+#### Interaction log
+
+Every interaction that get counted towards the INP metric will be added to the interaction log at the bottom of the panel.
+
+![Interaction log in DevTools](media/devtools-interaction-log.png)
+
+#### Interaction phases
+
+Phases for each interaction (including the one responsible for INP) are available by expanding the entry in the interaction log. Supports input delay, processing duration, and presentation delay.
+
+![Interaction phases in the interaction log](media/devtools-interaction-phases.png)
+
+#### LoAF attribution
+
+From the expanded interaction log, the "Local duration (ms)" heading will be clickable if there is LoAF attribution data available. Clicking it will log the data to the Console panel.
+
+![Additional LoAF attribution logged to the DevTools console from the interaction log](media/devtools-loaf-attribution.png)
+
+#### Layout shift log
+
+Adjacent to the interaction log is the layout shift log, which groups coincidental layout shifts into clusters and assigns a score to the cluster. The worst cluster corresponding to the page's CLS score is linked from the CLS section.
+
+![Layout shift log in DevTools](media/devtools-layout-shift-log.png)
+
+#### TTFB
+
+TTFB is available as a phase of LCP on the live metrics view as well as the trace view under the "LCP by phase" insight.
+
+![TTFB in DevTools trace view](media/devtools-ttfb.png)
+
+#### FCP
+
+FCP is shown as a marker in the trace view.
+
+![FCP marker in DevTools trace view](media/devtools-fcp.png)
+
+<h3 id="self">⚠️ Self-maintenance instructions</h3>
```

Comment:
This is the important bit we SHOULD have here (and again I missed this several times when skimming the README since it's using `<h3>` instead of `###`).

But it feels buried beneath the above. But I guess that's partially intentional.

Reply:
>This is the important bit we SHOULD have here (and again I missed this several times when skimming the README since it's using `<h3>` instead of `###`).

I just used the HTML tag to get the custom ID for easier deep-linking. Since we won't be maintaining this doc anymore the readability isn't a top concern.

>But it feels buried beneath the above. But I guess that's partially intentional.

Yeah, strong feelings about not pushing the self-maintenance option _before_ assisting users with the transition to DevTools. But let's use the other thread to talk about why you think we don't need that section.

Reply:
>  Since we won't be maintaining this doc anymore the readability isn't a top concern.

Yes that's totally a fair point. But it's also a reason why the generated anchor shouldn't change thought :-)

Anyway, not a big deal.

Reply:
>the generated anchor shouldn't change

All I'm saying is `#self` is a lot nicer to share than `#%EF%B8%8F-self-maintenance-instructions`

Reply:
Oh that is ugly! Didn't realise it didn't strip out emojis when auto generating the slug and converted them to that format instead. OK I'm convinced!